src/Microsoft.Health.Fhir.Api/Features/SMART/SmartClinicalScopesMiddleware.cs

```diff
@@ -53,9 +54,9 @@ public SmartClinicalScopesMiddleware(RequestDelegate next, ILogger<SmartClinical
 
             var authorizationConfiguration = securityConfigurationOptions.Value.Authorization;
 
-            _logger.LogInformation("Principal exists {Principal}", fhirRequestContextAccessor.RequestContext.Principal != null);
-            _logger.LogInformation("securityConfigurationOptions Value Enabled {SecutiryConfigurationOption}", securityConfigurationOptions.Value.Enabled);
-            _logger.LogInformation("Authorization configuration enabled {AuthorizationConfiguration} ", authorizationConfiguration.Enabled);
+            // _logger.LogInformation("Principal exists {Principal}", fhirRequestContextAccessor.RequestContext.Principal != null);
```

Comment:
nit: should we remove the commented lines

Reply:
I left those intentionally to use in future if any SMART related issues reported that needs additional logging.

Reply:
Done